Packs/CommonScripts/Scripts/ScheduleGenericPolling/ScheduleGenericPolling.py

```diff
@@ -16,14 +23,39 @@ def parseIds(idsArg):
     return str(idsArg)
 
 
+def should_run_with_guid():
+    """
+    The function verifies that the server has the right version in order to support
+     the stopScheduleEntry command and the add-on of the GUID to the Schedule command.
+    """
+    res_version = demisto.demistoVersion()
+    build_number = res_version.get('buildNumber')
+    platform = res_version.get('platform')
+
+    # conditions to add when the feature is supported in XSIAM:
+    # (platform == "x2" and is_demisto_version_ge(MINIMUM_XSIAM_VERSION) and int(
+    #     build_number) >= MINIMUM_BUILD_NUMBER_XSIAM)
+
+    return build_number != "REPLACE_THIS_WITH_CI_BUILD_NUM" and \
```

Comment:
What are we checking here?
```python
build_number != "REPLACE_THIS_WITH_CI_BUILD_NUM"
```

Reply:
On the local host the build number is this statement

Reply:
so we should remove it no?

Reply:
I will add a try/except mechanism to verify it will not fail.